neo/Core/TX/Transaction.py

```diff
@@ -775,6 +775,6 @@ def __init__(self, *args, **kwargs):
         self.Type = TransactionType.ContractTransaction
 
 
-class TXFeeError(Exception):
-    """Provide user-friendly feedback for transaction fee errors."""
+class TXError(Exception):
+    """Provide user-friendly feedback for transaction errors."""
```

Comment:
It seems like `TXError` is now obsolete. Is there any reason to keep it?

Reply:
I guess not.